src/algebra/big_operators/multiset.lean

```diff
@@ -78,6 +78,22 @@ lemma prod_nsmul (m : multiset α) : ∀ (n : ℕ), (n • m).prod = m.prod ^ n
 @[simp, to_additive] lemma prod_repeat (a : α) (n : ℕ) : (repeat a n).prod = a ^ n :=
 by simp [repeat, list.prod_repeat]
 
+@[to_additive]
+lemma prod_map_eq_pow_single [decidable_eq ι] (i : ι) (hf : ∀ i' ≠ i, i' ∈ m → f i' = 1) :
+  (m.map f).prod = (f i) ^ (m.count i) :=
```

Comment:
Arguably we can drop these parentheses. Not particularly obligatory to me, though:
```suggestion
  (m.map f).prod = f i ^ m.count i :=
```